### PR TITLE
Fixed imported module name

### PR DESCRIPTION
**Pull request by GlebKazantaev**
Fixed typo in python API test.

### CODE REVIEW

**Review by akuporos — APPROVED** [review]

**Comment by GlebKazantaev** [comment]
Waiting for @asuhov approval.

**Review by ilya-lavrenov — APPROVED** [review]

**Review by asuhov — APPROVED** [review]

**Pull request merged**
